drivers/iio/proximity/Kconfig

```diff
@@ -123,4 +123,9 @@ config VL53L0X_I2C
 	  To compile this driver as a module, choose M here: the
 	  module will be called vl53l0x-i2c.
 
+config VCNL3020
+	tristate "VCNL 3000"
```

Comment:
3000? A proper description needed. E.g. "Vishay Semiconductor VCNL3020 proximity sensor" or smth. like that.

Reply:
y, it should be changed + help section